src/app/project/documents/documents-tab.component.ts

```diff
@@ -466,6 +485,10 @@ export class DocumentsTabComponent implements OnInit, OnDestroy {
       queryModifiers['datePostedEnd'] = datePostedEnd;
     }
 
+    if (this.storageService) {
+      this.storageService.state[this.currentProject._id].tableParams = this.tableParams;
```

Comment:
Could this just be solved at the routing level without storing anything in state? If the filters are in the params, then using the back button on the browser should return to the previous page with them still in the url. If they navigate using a back button in the app then maybe we should update to handle the routing correctly?

Reply:
That was the first try, but I found a lot of the routing handles the params collections and the filter for UI/API stuff differently which was causing issues when paging. Using the state handler mitigated those issues. Basically, we "should" update the routing to handle this, but it will mean updating a lot of stuff everywhere... so this was a happy compromise

Reply:
Thanks for the explanation.